setup-homebrew/main.sh

```diff
@@ -1,49 +1,48 @@
-#/bin/bash
+#!/bin/bash
 
 set -euo pipefail
 
-if which brew &>/dev/null; then
```

Comment:
I think it's worth leaving around this code, personally, to handle self-hosted Linux workers and in case they remove Homebrew on Linux from the Linux worker on Actions again.

Reply:
Done